src/caches.jl

```diff
@@ -146,15 +143,8 @@ function OrdinaryDiffEq.alg_cache(
     measurement_model = make_measurement_model(f)
 
     # Initial State
-    initial_variance = ones(uElType, q + 1)
-    μ0 = uElType <: LinearAlgebra.BlasFloat ? Array{uElType}(calloc, D) : zeros(uElType, D)
-    Σ0 = PSDMatrix(
-        to_factorized_matrix(
-            FAC,
-            IsometricKroneckerProduct(d, diagm(sqrt.(initial_variance))),
-        ),
-    )
-    x0 = Gaussian(μ0, Σ0)
+    x0 = initial_distribution(prior)
```

Comment:
This is still not great :/ The initial distribution should really be part of the prior. But where is the diffusion then? So many questions.